### PR TITLE
Add `withLocale` to localize render view file

### PR DESCRIPTION
**Pull request by thenotsoft**
Merge after https://github.com/yiisoft/view/pull/199

### CODE REVIEW

**Comment by rustamwin** [comment]
Isn't possible to use `setLanguage`?

**Comment by thenotsoft** [comment]
@rustamwin no, setLanguage changes the language globally, I only need to change it locally.

**Review by rustamwin — APPROVED** [review]

**Review by samdark — APPROVED** [review]
Need a CHANGELOG line.

File: src/Mailer.php [hunk 1]
```diff
@@ -44,6 +44,20 @@ public function withTemplate(MessageBodyTemplate $template): self
...
     }
 
+    /**
+     * Returns a new instance with specified locale code.
+     *
+     * @param string $locale The locale code.
+     *
+     * @return self
+     */
+    public function withLocale(string $locale): self
```
Comment (vjik) [inline src/Mailer.php:54]: Need test on immutable.

**Review by terabytesoftw — APPROVED** [review]

**Pull request merged**